projects/fsastorefrontlib/src/cms-components/my-account/quote/quotes/quotes.component.ts

```diff
@@ -167,25 +166,18 @@ export class QuotesComponent implements OnInit, OnDestroy {
       });
       return quote;
     }, {});
+    this.selectedQuoteFromQuoteDetails();
   }
 
   protected setCategoryDropdown() {
+    const firstOption = this.translatePipe.transform('quote.allQuotes');
```

Comment:
It's better for translation to use `TranslatePipe` and avoid `TranslationService` and dealing with Observable data. TranslatePipe does the same thing and can be provided in multiple places since it is exported in the `I18nModule`.

Reply:
I would suggest we investigate this in more detail, because in official Spartacus documentation they cite as the best practice to use [translation service inside ts files](https://sap.github.io/spartacus-docs/i18n/#using-translations-in-typescript-code). Furthermore, there isn't any Spartacus code that uses TranslationPipe inside ts files.